Makefile

```diff
@@ -8,38 +8,89 @@ ifndef KIMAI_VERSION
   KIMAI_VERSION := master
 endif
 
+ifndef KIMAI_REPO
+	KIMAI_REPO := https://github.com/kevinpapst/kimai2.git
+endif
+
+ifndef KIMAI_IMG
+	KIMAI_IMG := kimai/kimai2
+endif
```

Comment:
I don't think the image names should be configurable. Official Kimai images are provided under the official name. 
Not sure what you want to achieve, but if you want to fork Kimai and/or the docker images, you are welcome to hack the hell out of your own fork. I don't see a reason to allow publishing the official images under any other name.

Reply:
just for development purposes.  i'm not tied to the notion, so i will remove this if its a blocker.